helm/mail/templates/configmap-logrotate.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.metrics.enabled }}
+{{- if .Values.metrics.logrotate.enabled }}
+{{- $chart := "mail" -}}
+{{- $labels := include (print $chart ".labels") . -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: logrotate-config
```

Comment:
Name should be based on chart name, otherwise you risk into clashes with existing configmaps in the same namespace or people deploying the same chart under a different name in the same namespace.